en/docs/WIP/configuring-api-security-audit.md

```diff
@@ -1,82 +1,87 @@
 
-    ![]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)
+     [![API token copy generated]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)
 
-6.  Create a collection by clicking on **API Collections** from the left navigation bar or by clicking on **+ New Collection** at the bottom of the left navigation.
+6.  Click either **API Collections** in the left navigation bar or click **+ New Collection** at the bottom of the left navigation to create a collection.
 
-    !!! note
-        A Collection in this context is a folder hosted on 42Crunch containing all the APIs that are to be audited.
+     A Collection in this context is a folder hosted on 42Crunch containing all the APIs that are to be audited.
 
-    ![]({{base_path}}/assets/img/Learn/collection-create-new.png)
+     [![create new collection]({{base_path}}/assets/img/Learn/collection-create-new.png)]({{base_path}}/assets/img/Learn/collection-create-new.png)
 
 7.  Copy the Collection ID from the URL of the browser as highlighted below and save it in a safe place for reference.
 
-    ![]({{base_path}}/assets/img/Learn/collection-copy-id.png)
+     [![copy collection id]({{base_path}}/assets/img/Learn/collection-copy-id.png)]({{base_path}}/assets/img/Learn/collection-copy-id.png)
 
-### Configure WSO2 API-M with the retrieved properties
+### Step 1.2 - Configure WSO2 API-M
 
-The API Token and Collection ID properties have to be placed inside the configuration files inside WSO2 API-M. There are two ways to do so.
+You need to add the API Token and Collection ID properties inside the configuration files in WSO2 API-M in order to configure WSO2 API-M with the retrieved properties. Use one of the following methods to carry out this process.
 
-#### Enabling the feature for all tenants
+#### Enable API security audit for all tenants
 
-Navigate to the `<API-M_HOME>/repository/conf/deployment.toml` file. Add the following configuration to the file and save the changes.
+1.  Navigate to the `<API-M_HOME>/repository/conf/deployment.toml` file. 
 
-``` bash tab="Format"
+2.  Add the following configuration to the file and save the changes.
+
+     ``` bash tab="Format"
     [security_audit]
     api_token="{api-token}"
     collection_id="{collection-id}"
     global=true
-```
+     ```
 
-``` bash tab="Example"
+     ``` bash tab="Example"
     [security_audit]
     api_token="c21404ea-p13x-1swq-013a-pur90605uiwl"
     collection_id="a5213vyo-6tre-560u-p04h-p0inb98i0gt1"
     global=true
-```
+     ```
 
-!!! tip
-    Set **global** property to **false** to disable the feature for all other tenants except the super tenant.
+    !!! tip
+        Set the **global** property to **false** to disable the feature for all other tenants except the super tenant.
 
-#### Enabling the feature for a single tenant
+#### Enable API security audit for a single tenant
```

Comment:
"**Audit API**" here as well

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/691/commits/38cee297b92e014c794b38f3e71091b8f9718df0